corgidrp/detector.py

```diff
@@ -506,10 +514,10 @@ def find_plateaus(streak_row, fwc, sat_thresh, plat_thresh, cosm_filter):
         return None
     
 def calc_sat_fwc(emgain_arr,fwcpp_arr,fwcem_arr,sat_thresh):
```

Comment:
The indentation level of this whole function seems to have gotten changed to 2 tabs. Can you change it back to 1 tab?

Reply:
^ @ramya-anche